free-3.md

```diff
@@ -10,34 +10,41 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Many students in CS70 are frusterated by the makeFiles the use to compile and run their code with all the correct dependencies. They may spend hours debugging trying to find what is wrong with their program, when really its just an issue with the makeFile. I would like to make a tool like sbt that works for C++ instead. This would recognize when a dependent file was updated and saved and then recompile and run. This language would be clearer and simpler to use, allowing students to focus their time and energy on their actual C++ code and make the debugging process faster and easier by allow them to make minor changes to their code, save, and then have everything automatically recompile and run.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for the same reasons that the DSL sbt is appropriate for making Scala programs easier to debug and rerun. This is an small, frequently used domain in which intuitive commands are needed to compile and run the C++ programs with all the right dependencies but there is not yet an easy to use tool.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+When Prof. Ben showed us sbt in class when he introduced Scala, my first thought was "Oh, why didn't we have this in CS70, this is so much easier than how we did it" as soon as someone asked if this was one of the DSLs hidden in the presentation, I realized that it was and that I could perhaps make a useful tool like that which would ease the frusteration of the CS70 students following in my footsteps. I love the practicality of this idea.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Simple build system for C++.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I would like to allow the user to simply type a short command into the commandLine and then have the program check for dependencies, newly saved file changes or errors, compile, and run the program. My users should be able to use only one or two broad commands to create a "large" program in which the compiler is executing quite a bit of code. Most build automation software is used in a similar way, so it would be easy for my users to adapt to using my language.
 
```

Comment:
You make a good point about other build automation softwares--I think planning your language so it's like other similar tools would make it more user friendly.